doc/Caliptra.md

```diff
@@ -415,21 +415,171 @@ The owner key, when represented in fuses or in the FMC's alias certificate, is a
+Caliptra certificates follow the X.509 v3 format described in RFC 5280. After vendor provisioning, Caliptra's certificate chain contains the following certificates:
+1. Provisioner CA (may be one or more certificates)
+2. IDevID
+3. LDevID
+4. Alias<sub>FMC</sub>
+5. Alias<sub>RT</sub>
+6. DPE
+
+After owner provisioning, an Owner CA may endorse the IDevID, LDevID, or Alias<sub>FMC</sub>. Owner CA provisioning is outside the scope of this specification.
+
+Caliptra generates the LDevID, Alias<sub>FMC</sub>, Alias<sub>RT</sub>, and DPE certificates. The vendor and optionally the owner generate all other certificates.
+
+### Provisioner CA
+Provisioner CA (pCA) is a set of one or more certificates issued by the vendor. The vendor is responsible for provisioning pCA to the SoC. Caliptra does not consume pCA. See [Reference 5](#ref-5) for guidance on pCA.
+
+### IDevID Certificate
+The vendor issues the IDevID certificate during SoC manufacturing. As part of [provisioning IDevID during manufacturing](#provisioning-idevid-during-manufacturing), Caliptra uses the UDS to derive the IDevID key pair and generate a CSR. The vendor's pCA uses the CSR to generate and sign the IDevID certificate.
+
+For IDevID to endorse LDevID, Caliptra requires the vendor to implement the following field values for the IDevID certificate:
+| Field                          | Sub Field   | Value
+| -------------                  | ---------   | ---------
+| Version                        | v3          | 2
+| Serial Number                  |             | First 20 bytes of SHA256 hash of IDevID public key
+| Validity                       | notAfter    | 99991231235959Z
+| Subject Name                   | CN          | Caliptra IDevID
+|                                | serialNumber | SHA384 hash of Subject public key
+| Subject Public Key Info        | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+|                                | Public Key  | DPE Alias Public Key value
+| Signature Algorithm Identifier | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+| Signature Value                |             | Digital signature for the certificate
+| KeyUsage                       | keyCertSign | 1
+| Basic Constraints              | CA          | TRUE
+|                                | pathLen     | 5
+
+Caliptra does not consume the IDevID certificate. Caliptra needs attributes of the IDevID certificate in order to generate the Authority Key Identifier extension for the LDevID and to populate the TCG Universal Entity ID (UEID) extension for Caliptra-generated certificates. The vendor must fuse these attributes into the IDevID attribute fuses for Caliptra to consume. The encoding of these attribute fuses are as follows:
```

Comment:
`encoding of these attribute fuses are as follows`-->`encoding of these attribute fuses is as follows`